src/routes/album/album.component.tsx

```diff
@@ -124,7 +140,15 @@ const Album = ({
 
             <div className='flex flex-col items-start justify-between sm:px-1 xsm:px-1 lg:pl-32 ms:px-4 xsm:ml-4 sm:ml-4 lg:ml-0 lg:mt-0 md:mt-2 sm:mt-4 xsm:mt-4'>
               <div>
-                <h1 className='text-lg'>{album.title}</h1>
+                <div className='flex flex-col'>
+                  <h1 className='text-lg flex flex-row'>{album.title}</h1>
+
+                  <div className='flex flex-row items-center justify-start w-12'>
+                    <span>{album.rate}</span>
```

Comment:
rate may be a number like 3.33333333333. You need to handle this case. .toFixed may help you